index.ftd

```diff
@@ -1,51 +1,1617 @@
+toc: $obj
+
+-- end: ftd.column
+
+-- end: ftd.column
+
+-- end: toc-instance
+
+
+
+
+
+
+
+
+
+
+-- component childrens:
+pr.toc-item toc:
+
+-- ftd.column:
+width: fill-container
+
+-- ftd.text:
+link if { childrens.toc.url != NULL }: $childrens.toc.url
+text: $childrens.toc.title
+role: $fonts.button-medium
+color: $inherited.colors.text
+opacity: 0.60
+
+-- ftd.column:
+if: {!ftd.is_empty(childrens.toc.children) }
+width: fill-container
+margin-top.px: 24
+spacing.fixed.px: 24
+
+-- child-of-childrens:
+$loop$: $childrens.toc.children as $obj
+toc: $obj
+
+-- end: ftd.column
+
+-- end: ftd.column
+
+-- end: childrens
+
+
+
+
+
+
+
+
+
+
+-- component child-of-childrens:
+pr.toc-item toc:
+
+-- ftd.column:
+width: fill-container
+padding-left.px: 10
+spacing.fixed.px: 24
+
+-- ftd.text:
+link if { child-of-childrens.toc.url != NULL }: $child-of-childrens.toc.url
+text: $child-of-childrens.toc.title
+role: $inherited.types.copy-small
+color: $inherited.colors.text
+color if { child-of-childrens.toc.is-active }: $inherited.colors.text-strong
+
+-- ftd.column:
+if: { !ftd.is_empty(child-of-childrens.toc.children) }
+width: fill-container
+
+-- child-of-childrens:
+$loop$: $child-of-childrens.toc.children as $obj
+toc: $obj
+
+-- end: ftd.column
+
+-- end: ftd.column
+
+-- end: child-of-childrens
+
+
+
+
+
+
+
+
+
+
+-- component h1:
+caption title:
+body body:
+optional string id: 
+
+-- ftd.column:
+id if { h1.id != NULL }: $h1.id
+width: fill-container
+spacing.fixed.px: 20
+region: h1
+
+-- ftd.text: $h1.title
+role: $inherited.types.heading-large
+color: $inherited.colors.text
+
+-- ftd.text:
+role: $inherited.types.copy-regular
+color: $inherited.colors.text
+opacity: 0.60
+
+$h1.body
+
+-- end: ftd.column
+
+-- end: h1
+
+
+
+
+
+
+
+
+
+
+-- component h2:
+caption title:
+optional body body:
+optional string id:
+
+-- ftd.column:
+id if { h2.id != NULL }: $h2.id
+width: fill-container
+padding-vertical.px: 24
+spacing.fixed.px: 18
+region: h2
+
+-- ftd.text: $h2.title
+role: $inherited.types.heading-small
+color: $inherited.colors.text
+
+-- ftd.text:
+if: { h2.body != NULL }
+role: $inherited.types.copy-large
+color: $inherited.colors.text
+opacity: 0.60
+
+$h2.body
+
+-- end: ftd.column
+
+-- end: h2
+
+
+
+
+
+
+
+
+
+
+-- component markdown:
+body body:
+
+-- ftd.column:
+margin-bottom.px: 20
+
+-- ftd.text:
+role: $inherited.types.copy-large
+color: $inherited.colors.text
+opacity: 0.60
+
+$markdown.body
+
+-- end: ftd.column
+
+-- end: markdown
+
+
+
+
+
+
+
+
+
+
+-- component list-item-bold:
+caption title:
+
+-- ftd.row:
+width: fill-container
+spacing.fixed.px: 8
+margin-bottom.px: 8
+
+-- ftd.text: &#x2022;
+role: $inherited.types.label-large
+style: bold
+color: $inherited.colors.shadow
+
+-- ftd.text: $list-item-bold.title
+role: $inherited.types.label-large
+style: bold
+color: $inherited.colors.shadow
+
+-- end: ftd.row
+
+-- end: list-item-bold
+
+
+
+
+
+
+
+
+
+
+-- component code:
```

Comment:
@MeenuKumari28 this component is already imported inside fastn.ftd remove all typography components from this file